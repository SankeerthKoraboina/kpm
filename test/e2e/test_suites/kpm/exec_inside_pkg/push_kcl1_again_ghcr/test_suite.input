kpm push oci://ghcr.io/kcl-lang/helloworld