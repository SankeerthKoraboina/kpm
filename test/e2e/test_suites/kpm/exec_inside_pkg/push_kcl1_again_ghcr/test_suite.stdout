package 'kcl1' will be pushed